js-packages/@prestojs/ui/src/FieldWidgetInterface.ts

```diff
@@ -58,6 +58,8 @@ export interface WidgetProps<FieldValue, T extends HTMLElement> {
 export interface RangedWidgetProps<FieldValue, T extends HTMLElement, P> {
     lowerInput: P;
     upperInput: P;
+    // the "input" from default form
+    input: InputProps<FieldValue, T> & { value?: Record<string, FieldValue> };
```

Comment:
can you document each key here please

should be format

```
/**
 * lowerInput docs
 */
lowerInput: P
/**
  * upperInput docs
  */
upperInput: P
...
```

you can run `cd doc-site && node extract-docs.js` and visit `localhost:4000` to see what docs look like when rendered